front/src/pages/login/Login.jsx

```diff
@@ -1,3 +1,157 @@
+import { Fragment } from 'react';
+import Logo from '../../components/logo/Logo';
+import styles from './Login.module.css';
+
 export default function Login() {
-  return <h1> Login </h1>;
+  return (
+    <Fragment>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          width: 1284,
+          height: 769,
+        }}
```

Comment:
a gente poderia ter usado classes ao invés de inline styles :cry: 

https://stackoverflow.com/questions/3142710/whats-the-difference-between-inline-styles-vs-classes